src/item_factory.cpp

```diff
@@ -1888,6 +1888,21 @@ void Item_factory::load_magazine( JsonObject &jo, const std::string &src )
     }
 }
 
+void Item_factory::load( islot_battery &slot, JsonObject &jo, const std::string & )
```

Comment:
Should it be `const std::string &src` ?

Reply:
That would actually cause a compiler warning since the `const std::string &` parameter isn't used.

Reply:
I'll just stick upvoting - I know I was stepping out of my bounds :P

Reply:
No worries!